### PR TITLE
[interp] run regression with all combinations of available optimizations

### PR DESCRIPTION
**Pull request by lewurm**
Like we do for the JIT.

### CODE REVIEW

File: mono/mini/driver.c [hunk 1]
```diff
@@ -326,6 +326,50 @@ opt_sets [] = {
...
+	INTERP_OPT_INLINE,
+	INTERP_OPT_CPROP,
+	INTERP_OPT_SUPER_INSTRUCTIONS,
+	INTERP_OPT_INLINE | INTERP_OPT_CPROP,
+	INTERP_OPT_INLINE | INTERP_OPT_SUPER_INSTRUCTIONS,
+	INTERP_OPT_CPROP | INTERP_OPT_SUPER_INSTRUCTIONS,
+	INTERP_OPT_INLINE | INTERP_OPT_CPROP | INTERP_OPT_SUPER_INSTRUCTIONS,
+};
+
+static const char*
```
Comment (jaykrell) [inline mono/mini/driver.c]: Two const please -- one for the chars, one for the pointers.

File: mono/mini/driver.c [hunk 2]
```diff
@@ -326,6 +326,50 @@ opt_sets [] = {
...
+	return interp_opflags_names [i];
+}
+
+static char*
+interp_opt_descr (guint32 flags)
+{
+	GString *str = g_string_new ("");
+	int i, need_comma;
+
+	need_comma = 0;
```
Comment (jaykrell) [inline mono/mini/driver.c]: gboolean?

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewurm** [comment]
@monojenkins squash

**Pull request merged**